package/libpiksi/libpiksi/src/util.c

```diff
@@ -21,14 +21,28 @@
 #include <libpiksi/util.h>
 #include <libpiksi/logging.h>
 
+/* from pfwp init.c */
+#define IMAGE_HARDWARE_INVALID 0xffffffff
+#define IMAGE_HARDWARE_UNKNOWN 0x00000000
+#define IMAGE_HARDWARE_V3_MICROZED 0x00000001
+#define IMAGE_HARDWARE_V3_EVT1 0x00000011
+#define IMAGE_HARDWARE_V3_EVT2 0x00000012
+#define IMAGE_HARDWARE_V3_PROD 0x00000013
+
 #define SBP_SENDER_ID_FILE_PATH "/cfg/sbp_sender_id"
 #define DEVICE_UUID_FILE_PATH   "/cfg/device_uuid"
 
+#define DEVICE_HARDWARE_REVISION_FILE_PATH  "/factory/hardware"
+#define DEVICE_HARDWARE_REVISION_MAX_LENGTH (64u)
+#define DEVICE_HARDWARE_VERSION_FILE_PATH   "/factory/hardware_version"
+#define DEVICE_HARDWARE_VERSION_MAX_LENGTH  (64u)
+
 #define DEVICE_DURO_EEPROM_PATH "/cfg/duro_eeprom"
 #define DEVICE_DURO_MAX_CONTENTS_SIZE (128u)
 #define DEVICE_DURO_ID_STRING "DUROV0"
```

Comment:
Can we change this to just check for DURO, to potentially allow us to programmatically tell the difference between DUROV0, DUROV1, DUROV2, etc? down the line?

Reply:
I think we can cross that bridge when we get there